Automattic/jetpack
# [Plugin] Backup: Change Backup related info blocks to only display with Backup plan

#### Changes proposed in this Pull Request:
Info blocks that appear under the Hero area that pertain to Backups will now only appear when the site's plan has backup capabilities.

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Install the plugin on a site and ensure the Activity log and 'Where are my backups stored' blocks appear with an active Backup plan
<img width="1394" alt="Screen Shot 2021-07-14 at 2 38 55 PM" src="https://user-images.githubusercontent.com/1992658/125674984-07758791-e2f0-4933-b7ce-9dbd37ee505f.png">
* Ensure the Activity log and 'Where are my backups stored' blocks do not appear without an active Backup plan
<img width="1433" alt="Screen Shot 2021-07-14 at 2 33 25 PM" src="https://user-images.githubusercontent.com/1992658/125674525-70995f02-93fc-4fcf-b3c8-a286787a4678.png">
